src/script/components/TextInput/TextInput.tsx

```diff
@@ -96,7 +98,9 @@ const TextInput: React.ForwardRefRenderFunction<HTMLDivElement, UserInputProps>
         </span>
       )}
 
+      {/* eslint-disable */}
```

Comment:
humm this sounds bad

Reply:
haha, there's a reason I flagged it as "Needs review", thanks for suggesting a way forward :)

Reply:
just disable one rule